index.html

```diff
@@ -0,0 +1,244 @@
+                css
+              </li>
+              <li>
+                javascript
+              </li>
+            </ul>
+          </div>
+          <div>
+            <button> See Project</button>
+          </div>
+        </div>
+
+        <div class="grid column row">
+          <div class="four"></div>
+          <div>Multi-Post Stories</div>
+          <div>
+            <div>CANOPY</div>
+            <ul class="flex">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <div>
+            A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+          </div>
+          <div>
+            <ul class="flex">
+              <li>
+                html
+              </li>
+              <li>
+                css
+              </li>
+              <li>
+                javascript
+              </li>
+            </ul>
+          </div>
+          <div>
+            <button> See Project</button>
+          </div>
+        </div>
+
+        <div class="about grid row2 column">
+          <section>
+            <h1>About<br>Myself</h1>
+            <p>
+              Hello I’m a software developer! 
+              I can help you build a product,
+              feature or website Look through 
+              some of my work and experience!
+              If you like what you see and have 
+              a project you need coded, 
+              don’t hestiate to contact me.
+            </p>
+            <div class="connect">LET'S CONNECT</div>
+            <div>
+              <ul class="flex">
+                 <li>
+                   <a href="#"><img src="icons/space.svg" alt="icon"></a>
+                 </li>
+                 <li>
+                   <a href="#"><img src="icons/git.svg" alt="icon"></a>
+                 </li>
+                 <li>
+                   <a href="#"><img src="icons/chi.svg" alt="icon"></a>
+                 </li>
+                 <li>
+                   <a href="#"><img src="icons/twitter.svg" alt="icon"></a>
+                 </li>
+              </ul>
+            </div>
+            <button>
+              Get my resume
+            </button>
+          </section>
+        </div>
+
+          <div class="grid column lan">
+  
+            <ul>
+                <li>
+                    <p>Languages</p>
+                    <!--<div><img src="images/ad.svg" alt="ad"></div>-->
+                </li>
+                <li>
+                    <div><img src="images/js.svg" alt="js"></div>
+                    <p class="diff">JavaScript</p>
+                </li>
+                <li>
+                    <div><img src="images/html.svg" alt="html"></div>
+                    <p class="diff">HTML</p>
+                </li>
+                <li>
+                    <div><img src="images/css.svg" alt="css"></div>
+                    <p class="diff">CSS</p>
+                </li>
```

Comment:
the Text of your languages is not aligned correctly, please try to stick to the Figma design and Kindly change it

- [ ] Your design |
![image](https://user-images.githubusercontent.com/61790587/164925064-acbad5d4-e8f2-42e9-9711-19b5ebbc6a2d.png)

- [ ] Figma design 
![image](https://user-images.githubusercontent.com/61790587/164925854-a794d98d-2cd2-4328-a636-a2b2518b8cec.png)